docs/website/content/integrations/regocpp.md

```diff
@@ -10,4 +10,4 @@ tutorials:
 code:
 - https://github.com/microsoft/rego-cpp
 ---
-The rego-cpp project is a Rego compiler and runtime implemented in C++.  As such, it provides the ability to integrate Rego natively into a wider range of languages. We currrently support C, C++, Rust, and Python, and are largely compatible with v0.55.0 of the language.
\ No newline at end of file
+The rego-cpp project is a Rego compiler and runtime implemented in cross-platform C++.  As such, it provides the ability to integrate Rego natively into a wider range of languages on most platforms. In addition to CMake FetchContent support for C and C++, we also distribute a Rust crate and a Python wheel. We support v0.68.0 of the rego language as well as full v1.0 compatibility.
```

Comment:
🤔 I'm assuming you're following the latest OPA release? If so, I would rephrase this, or you'd need to update the docs every month... also, 

> v0.68.0 of the rego language

sounds a bit wrong to me. The Rego language (capital R) isn't versioned as OPA is... so, I suppose you support "Rego as of OPA v0.68.0" or "Rego, as of the latest OPA release"

Reply:
Yeah, I can probably change the wording to be latest release to avoid having to update this regularly.